src/show/components/Navbar/index.js

```diff
@@ -4,8 +4,8 @@ import { Fragment, useState } from 'react';
 import { TbStairsUp } from 'react-icons/tb';
```

Comment:
Seems like you skiped default props and prop types for this file. Are there others?

Reply:
i don't have proptypes in navbar